### PR TITLE
Add field for whether dataset contains SSN

### PR DESCRIPTION
**Pull request by mattboehm**

### CODE REVIEW

**Comment by mattboehm** [comment]
@tanderegg if this can be merged by tomorrow, we can include it in a release to be deployed before/after the sprint meeting.

File: ckanext/cfpb_extrafields/exportutils.py [hunk 1]
```diff
@@ -107,5 +108,8 @@ def to_csv(data, fields, fieldmap=tuple(FIELDS)):
     writer = csv.DictWriter(output, [f[0] for f in fields], extrasaction="ignore")
     writer.writerow(dict(fieldmap))
     for result in data:
-        writer.writerow(flatten(result))
+        row = flatten(result)
+        #If blank, default to "No"
+        row["privacy_contains_ssn"] = row.get("privacy_contains_ssn") or "No"
```
Comment (tanderegg) [inline ckanext/cfpb_extrafields/exportutils.py]: @mattboehm Minor tweak, but might as well do it as row.get("privacy_contains_ssn", "No")?  One nice thing Ruby has is ||=, no equivalent to that in Python :)

File: ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html [hunk 2]
```diff
@@ -275,6 +275,10 @@
         <td class="dataset-details">{{ pkg_dict.privacy_contains_pii}} </td>
         </tr>
         {% endif %}
+        <tr>
+        <th scope="row" class="dataset-label">{{ _("privacy: contains SSN") }} </th>
+        <td class="dataset-details">{{ pkg_dict.privacy_contains_ssn|default('no')}} </td>
```
Comment (tanderegg) [inline ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:280]: @mattboehm The default above was "No", shouldn't it be the same here?

**Review by tanderegg — CHANGES_REQUESTED** [review]
Some minor updates, let me know when done and I will merge.

File: ckanext/cfpb_extrafields/exportutils.py (hunk 1 above)
Comment (mattboehm) [reply ckanext/cfpb_extrafields/exportutils.py]: I also wanted to convert empty strings (and other falsy values) to "No".

Comment (tanderegg) [reply ckanext/cfpb_extrafields/exportutils.py]: Ah, never mind then!

File: ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html (hunk 2 above)
Comment (tanderegg) [reply ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:280]: @mattboehm Does this change need to be made?  If not I will just merge, thanks!

Comment (mattboehm) [reply ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:280]: @tanderegg Sorry, wanted to look into this further before responding. Unfortunately, all these yes/no fields have inconsistent casing in the CSV export, so we may want a separate ticket to fix that (probably due to edits made at different times or DIG import vs manual setting). The lowercase "no" is consistent with how other fields are displayed on this page so if anything, I'd change the capitalized one, but for the time being, I think we could merge this as-is.
screenshot of export

Comment (mattboehm) [reply ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:280]: Actually, lowercased it in the export as this is marginally more consistent.

Comment (tanderegg) [reply ckanext/cfpb_extrafields/templates/package/snippets/additional_info.html:280]: Agreed, can you create an issue to track this in Github?  Will merge now.

**Review by tanderegg — APPROVED** [review]

**Pull request merged**